docs/docs/tutorial/contentful/index.mdx

```diff
@@ -0,0 +1,311 @@
+
+```diff
+// package.json
+
+{
+  ...
+  "scripts": {
++   "develop": "gatsby develop",
++   "build": "gatsby build",
++   "serve": "gatsby serve",
++   "clean": "gatsby clean",
+    ...
+  },
+  "dependencies": {
+    "gatsby": "^4.x.x",
+    "gatsby-plugin-image": "^2.x.x",
+    "gatsby-source-contentful": "^7.x.x",
+    "react": "^18.x.x",
+    "react-dom": "^18.x.x"
+  }
+}
+```
+
+- develop: The develop script starts the Gatsby local development server.
+- build: The build script statically builds your Gatsby site.
+- serve: The serve script allows you to preview your build locally.
+- clean: The clean script removes the `public` and `.cache` directories.
+
+### 5. Contentful Space API Keys
+
+In the Contentful UI navigate to Settings > API Keys. Here you will find the Space ID and Content Delivery API access token.
+
+If you don’t already have a Space created, you can [reference the Contentful docs](https://www.contentful.com/help/contentful-101/#step-2-create-a-space).
+
+### 6. gatsby-config.js
+
+```javascript
+// gatsby-config.js
+
+module.exports = {
+  plugins: [
+    ...{
+      resolve: "gatsby-source-contentful",
+      options: {
+        spaceId: "...",
+        accessToken: "...",
+      },
+    },
+    "gatsby-plugin-image",
+  ],
+}
+```
+
+_For production sites you should add these as environment variables. There's more information in the docs about how to do this: [Environment Variables](https://www.gatsbyjs.com/docs/how-to/local-development/environment-variables/)._
+
+### 7. Create Content Type
+
+In the Contentful UI navigate to Content Model and click add type. To start with, create one called Page.
+
+### 8. Add Fields to Content Type
+
+With a Content type created, you can now add fields.
+
+Add the following fields to the Page.
+
+- slug: The slug field is required and is of type Text, and Short text.
+- title: This title field is required and is of type Text, and Short text.
+- description: This description field is required and is of type Text, and Long text.
+
+Fields names are case sensitive. Double check all field names are lowercase.
+
+You can read more about Contentful's [Field types in the Contentful doc](https://www.contentful.com/help/contentful-glossary/#fields).
+
+### 9. Add Content to Fields
+
+In the Contentful UI navigate to Content to create a new page Page. Add content for the slug, title and description. When you’re ready, publish your changes.
+
+Add the following content to create an Index or Home page.
+
+- slug: Set this to / for the index page of a site.
+- title: Add a title of the page.
+- description: Add a short description for the page.
+
+Repeat the same steps to create a Not Found page.
+
+- slug: Set this to /404 for the not found page of a site.
+- title: Add a title of the page.
+- description: Add a short description for the page.
+
+### 10. GraphiQL Explorer
```

Comment:
Would you consider Including Images in this session for more clarity?

Reply:
Hey @SauveJeanLuc they'll be a video at the top of the docs explaining each of the steps with a voiceover and closed captions.